app/models/ncr/work_order.rb

```diff
@@ -24,21 +26,81 @@ class WorkOrder < ActiveRecord::Base
     validates :building_number, presence: true
     validates :office, presence: true
 
-    # Ignore values in these fields, depending on the expense type
-    def ignore_fields
-      case self.expense_type
+    def set_defaults
+      self.not_to_exceed ||= false
+      self.emergency ||= false
+    end
+
+    # @todo - this is an awkward dance due to the lingering Cart model. Remove
+    # that dependence
+    def create_cart(approver_email, description, requester)
```

Comment:
`#create_ASSOCIATION_NAME` is reserved in ActiveRecord, so might be confusing, even if we're not using the other form at the moment. Mind changing the name?